packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -299,6 +302,7 @@ const mapState = ({
 const mapDispatch = {
   onCreateRegularIndex,
   onCreateSearchIndex,
+  onChangeIndexView: changeIndexView,
```

Comment:
We can call this method directly from the segmented control and remove `onChangeIndexesSegment`